WORKSPACE

```diff
@@ -1,14 +1,36 @@
 workspace(name = "com_github_istio_mixer")
 
 git_repository(
+    name = "org_pubref_rules_protobuf",
```

Comment:
what is the need to pull this out of `adapter_author_deps.bzl`? Can we change the ordering in a different way, so as to support external repo imports better?